credentials/credentials.go

```diff
@@ -50,6 +51,49 @@ type PerRPCCredentials interface {
 	RequireTransportSecurity() bool
 }
 
+// SecurityLevel defines the protection level on an established channel.
```

Comment:
Throughout: please don't use "channel".  gRPC-Go does not use that term.  You can use "connection" for an individual connection or "ClientConn" for a `grpc.ClientConn` connected to potentially many backends.

Reply:
Done in the next commit.